data/human/ships.txt

```diff
@@ -435,13 +435,13 @@ ship "Bastion"
 	attributes
 		category "Medium Warship"
 		"cost" 3860000
-		"shields" 7500
-		"hull" 4700
+		"shields" 9000
+		"hull" 5500
```

Comment:
```suggestion
		"shields" 8600
		"hull" 5300
```
Gives room for the Bulwark to maintain its current proportional advantage in HP over the Bastion without pushing into HW territory.